zookeeper-server/src/test/java/org/apache/zookeeper/test/ObserverMasterTest.java

```diff
@@ -112,15 +109,29 @@ private PortForwarder setUp(final int omProxyPort) throws IOException {
 
         OM_PORT = PortAssignment.unique();
 
-        String quorumCfgSection =
-                "server.1=127.0.0.1:" + (PORT_QP1)
-                        + ":" + (PORT_QP_LE1) + ";" +  CLIENT_PORT_QP1
-                        + "\nserver.2=127.0.0.1:" + (PORT_QP2)
-                        + ":" + (PORT_QP_LE2) + ";" + CLIENT_PORT_QP2
-                        + "\nserver.3=127.0.0.1:"
-                        + (PORT_OBS)+ ":" + (PORT_OBS_LE) + ":observer" + ";" + CLIENT_PORT_OBS;
+        String quorumCfgSection = "server.1=127.0.0.1:"
```

Comment:
it is worth reverting
can you take a look to all this similar cases in tests ?
I won't add any other comments on this kind of issue

Reply:
Yes. It requires another pass about line break things. I will take care of it.